tests/expr/unary_test.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+import narwhals as nw
+from tests.utils import compare_dicts
+
+
+def test_unary(constructor: Any) -> None:
+    data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.0, 8, 9]}
+    result = (
+        nw.from_native(constructor(data))
```

Comment:
I think it should be safe to run with lazy constructor as well

```suggestion
def test_unary(constructor_with_lazy: Any) -> None:
    data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.0, 8, 9]}
    result = (
        nw.from_native(constructor_with_lazy(data))
```